File: baekjoon/bronze/level1/test_1934/Main.java
package bronze.level1.test_1934;

import java.io.*;
import java.util.stream.Stream;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));

        int n = Integer.parseInt(br.readLine());

        for (int i = 0; i < n; i++) {
            int[] input = Stream.of(br.readLine().split(" ")).mapToInt(Integer::parseInt).toArray();
            int result = input[0] * input[1] / gcd(input[0], input[1]);
            bw.write(String.valueOf(result));
            bw.newLine();
        }
        bw.close();
    }

    public static int gcd(int a, int b) {
        if (b == 0) return a;
        return gcd(b, a % b);
    }
}

File: baekjoon/bronze/level1/test_1934/kotlin/Main.kt
package baekjoon.bronze.level1.test_1934.kotlin

fun main() {
    val n = readln().toInt()

    for (i in 0 until n) {
        val num = readln().split(" ").map { it.toInt() }
        val lcm = lcm(num[0], num[1])
        println(lcm)
    }
}

fun gcd(a: Int, b: Int): Int {
    return if (b == 0) a else gcd(b, a % b)
}

fun lcm(a: Int, b: Int): Int {
    return a / gcd(a, b) * b
}
